app/styles/crate.scss

```diff
@@ -118,12 +118,40 @@
     }
 }
 
+.crate-link {
+    display: inline-block;
+}
+
+@media only screen and (max-width: 639px) {
+    .white-rows div {
+        margin: auto auto 0.2rem auto;
+        text-align: center;
+        .info > [data-test-crate-link] {
```

Comment:
Hmm, my understanding is that these `data-test` attributes are only for the javascript tests to grab onto, and if we want to style things, we should add more classes for that purpose. Would you be willing to make that change?

Reply:
Absolutely. I’ll get back to you.

Reply:
bit late, but wanted to confirm: the `data-test-` attributes are stripped out from production builds so we must not rely on them in the CSS files or any other production code

Reply:
Yes. It has been removed. Please see the updated code. Thanks!